auth-web/src/components/auth/common/ContactUsTooltip.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-tooltip top max-width="450px" light content-class="tooltip" v-model="showTip">
```

Comment:
Content class should be a more specific name for this component so that it can be easily selected in the DOM for testing. Unless this is the only tooltip?

Reply:
This is only the tool tip. I did consider adding an Prop to the tooltips that will be the ID of the parent, passed into the tooltip so we can attach it where use it. 

I forgot to mention. This ToolTip still had some outstanding information and isn't currently implemented. Hence why there is incomplete data as mentioned below :) This is just something we will need once we have the info regarding what we want for mvp.

Reply:
Sounds like you're on top of this already.

My 2c: name the class "contact-us-tooltip" ?

Reply:
Agreed! Done in next commit